<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>561</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 30px;
border: 2px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.5 rgb(41, 115, 0), stop: 0.5 rgb(41, 115, 0));</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>541</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 1.1 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));
border-radius: 30px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
border: transparent;
color: gold;</string>
      </property>
      <property name="text">
       <string>Harga</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtJenisService">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
border: transparent;
color: gold;</string>
      </property>
      <property name="text">
       <string>Jenis Service</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtHarga">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
border: transparent;
color: gold;</string>
      </property>
      <property name="text">
       <string>Kode Service</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClear">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtMerek">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
border: transparent;
color: gold;</string>
      </property>
      <property name="text">
       <string>Merek</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSimpan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;ui/binkaibulet.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(&quot;icons/icon.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnHapus">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_14">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>21</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid gold;
background-color: gold;
color: rgb(85, 255, 0);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="txtKodeSV">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid gold;
background-color: rgb(13, 39, 0);
border-radius: 10px;
padding: 0 8px;
color: gold;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCari">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.4 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));
color: gold;</string>
       </property>
       <property name="text">
        <string>cari</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QTableWidget" name="gridService1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>561</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
gridline-color: rgb(85, 255, 0);</string>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent;
background: transparent;
color: gold;</string>
     </property>
     <property name="text">
      <string># Data Service #</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_20">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_18">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>130</y>
      <width>41</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px 40px;
float: left;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.5 rgb(41, 115, 0), stop: 0.5 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_13">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>41</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px 40px;
float: left;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.5 rgb(41, 115, 0), stop: 0.5 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_10">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_21">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_24">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_11">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_15">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>540</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_17">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_25">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_26">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_27">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8;
border: 5px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_28">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 13;
border: 2px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 13;
border: 2px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgb(85, 255, 0), stop: 0.8 rgb(41, 115, 0), stop: 1.0 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_29">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>401</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 200;
border: 15px solid rgb(85, 255, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 1 rgb(85, 255, 0), stop: 1.2 rgb(41, 115, 0), stop: 0.9 rgb(41, 115, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;icons/icon.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_29</zorder>
   <zorder>frame_28</zorder>
   <zorder>frame_12</zorder>
   <zorder>frame_13</zorder>
   <zorder>frame_8</zorder>
   <zorder>frame</zorder>
   <zorder>frame_9</zorder>
   <zorder>frame_20</zorder>
   <zorder>frame_18</zorder>
   <zorder>frame_10</zorder>
   <zorder>frame_21</zorder>
   <zorder>frame_24</zorder>
   <zorder>frame_17</zorder>
   <zorder>frame_15</zorder>
   <zorder>frame_11</zorder>
   <zorder>frame_26</zorder>
   <zorder>frame_25</zorder>
   <zorder>frame_27</zorder>
   <zorder>frame_16</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
